queue_services/entity-filer/tests/unit/filing_processors/test_continuation_out.py

```diff
@@ -0,0 +1,62 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Unit Tests for the Continuation Out filing."""
+import copy
+import random
+
+from legal_api.models import Business, Filing
+
+from registry_schemas.example_data import CONTINUATION_OUT, FILING_TEMPLATE
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors import continuation_out
+from tests.unit import create_business, create_filing
+
+async def test_worker_continuation_out(app, session):
```

Comment:
please update this test to include checks for any updates you've made to the continuation out processor

Reply:
Yes, I will

Reply:
Hi @argush3 
I think I implemented all cases.
Please kindly review it again.